README.md

```diff
@@ -40,15 +40,21 @@ extra seed data required.**
 
 ---
 
+
+### Set up database
+
 Setup the database connection:
 
 ```sh
 $ cp config/example.database.yml config/database.yml
 ```
-
 Be sure to remove or modify the `username` attribute.
 
-Make sure Elasticsearch is running.
+Download a backup of the Production DB and load it into your local DB. Follow the instructions from step 4 in this wiki: https://moneyadviceserviceuk.atlassian.net/wiki/display/RRAD/FCA+Import+-+Technical+Help
+
+### Set up Elasticsearch
+
+Make sure Elasticsearch is running. After starting it, if you navigate to http://localhost:9200/ the version number should be 1.7.x
```

Comment:
This is a great addition. Should we try to make this visually jump out as important somehow?